inventory/routes/concepts/inter_warehouse.rst

```diff
@@ -1,131 +1,128 @@
 =====================================
-How to do inter-warehouses transfers?
+Proceed to inter-warehouses transfers
 =====================================
 
-If you own different warehouses you might want to transfer goods from
-one warehouse to the other. This is very easy thanks to the inventory
-application in Odoo.
+When owning several warehouses, you might need to transfer goods from one warehouse to another.
+Proceeding to such behavior is called *inter-warehouses transfers*. If this action is physically
+easy to perform, the administrative part can be arduous. Fortunately, Odoo comes with an intuitive
+flow helping save time when registering those transfers.
 
 Configuration
 =============
 
-First of all you have to select the multi locations option. Go to
-:menuselection:`Configuration --> Settings` in the **Inventory application**.
-Then tick the **Manage several locations per
-warehouse** option. Please don't forget to **apply** your changes.
+First of all, go to :menuselection:`Configuration --> Settings` and enable **Storage Location**.
+Then, hit save.
 
-.. image:: media/inter01.png
+.. image:: inter_warehouse/feature-storage-location.png
    :align: center
+   :alt: Image of the storage location feature being enabled.
 
 .. tip::
-   This option should also be ticked if you wish to manage different
-   locations and routes in your warehouse.
+   If you want to manage several routes within your warehouses, then tick the **Multi-Step Routes**
+   and check :doc:`use-routes`.
 
 Creating a new warehouse
 ========================
 
-The next step is to create your new warehouse. In the Inventory application
-click on :menuselection:`Configuration --> Warehouse Management --> 
-Warehouses`. You are now able to create your warehouse by clicking on
-**Create**.
+The next step is to create your new warehouse. To do so, go to :menuselection:`Configuration -->
+Warehouse Management --> Warehouses`. Once there, click on **Create**.
 
-Fill in a **Warehouse Name** and a **Short Name**. The short name is 5
-characters maximum.
+Fill in a **Warehouse Name** and a **Short Name**. The short name is 5 characters maximum.
```

Comment:
Jonathan told me: "We decided to write in American English, therefore we use 'fill out' instead of 'fill in'."